invenio_communities/views/communities.py

```diff
@@ -22,7 +22,7 @@
 from invenio_communities.proxies import current_communities
 
 from ..communities.resources.ui_schema import TypesSchema
-from ..members.records.api import Member
```

Comment:
This removed line isn't on master so I'm confused

https://github.com/carlinmack/invenio-communities/blob/master/invenio_communities/views/communities.py

I removed it as `Member` isn't used